proto/index.proto

```diff
@@ -41,6 +41,15 @@ message IndexResponse {
   context.ResponseContext response_context = 1;
 }
 
+message RepoStatusRequest {
+  string namespace = 1;
+  string repo_url = 2;
+}
+
+message RepoStatusResponse {
+  string last_indexed_commit_sha = 1;
+}
```

Comment:
can you add request_context and response_context to these protos?

Reply:
done